### PR TITLE
Separate `pytest` settings for running plugin tests with Travis vs. Jenkins

### PR DESCRIPTION
**Pull request by kvfairchild**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -37,6 +37,18 @@ requires = [
...
+################################################################
+#### mypy type-checking config ####
+################################################################
+
+[tool.pytest.ini_options]
+markers = [
+    "memory_intense",
+    "travis_slow",
+    "slow",
+    "private_access"
```
Comment (mschrimpf) [inline pyproject.toml:49]: can we include descriptions? see https://github.com/brain-score/brain-score/pull/413/files

**Review by mschrimpf — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (shehadak) [reply pyproject.toml:49]: Does the following look reasonable?

```suggestion
    "memory_intense: tests requiring more memory than is available in the Travis sandbox (currently 3 GB)", # see https://docs.travis-ci.com/user/common-build-problems/#my-build-script-is-killed-without-any-error
    "travis_slow: tests running for more than 10 minutes without output (which leads Travis to error)",
    "slow: tests leading to runtimes that are not possible on the OpenMind cluster (>1 hour per test)",
    "private_access: tests that require access to a private resource, such as assemblies on S3 (note that Travis pull request builds can not have private access)",
```

Comment (kvfairchild) [reply pyproject.toml:49]: I made a suggestion that clarifies `private_access` [on the referenced PR](https://github.com/brain-score/brain-score/pull/413/files); Travis PR builds _can_ have private access as long as they do not originate from a fork. If that can be updated here too, then this works for me.

Comment (shehadak) [reply pyproject.toml:49]: Updated the suggestion to use the description from the referenced PR. If all is good, feel free to commit the suggestion :)

**Pull request merged**